### PR TITLE
[RFC] messages: Update common instances of Vim to Nvim

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by philix** [comment]
Is this still a WIP? Do you plan to add more changes?

**Comment by Pyrohh** [comment]
It looks like I rebased and fixed all merge conflicts a week ago, but never pushed :smile:

Also, an earlier PR I did seems to have already dealt the need for the `docs` portion of the PR, so this is now about 10 times smaller.

**Comment by Pyrohh** [comment]
@justinmk @philix what do you think about me dropping 5c61903366b5d573d832c94dc794bef94bc496a0? IMO, the benefit is almost nil, and it's not like those decrepit runtime files are areas of much activity.

**Comment by philix** [comment]
@Pyrohh why do you want to drop it? It's a good idea to remove references to `compatible`.

**Comment by Pyrohh** [comment]
@philix nocp always evaluates to true, so yes the removed checks are indeed useless, but I think they might make merging runtime files more difficult (although barely). I'm not really sure though...

**Comment by philix** [comment]
>  but I think they might make merging runtime files more difficult (although barely)

You're right. It's not worth the hassle.

**Comment by Pyrohh** [comment]
I dropped that commit, I'll merge this after tests pass (if no-one objects).

File: src/nvim/fileio.c [hunk 1]
```diff
@@ -724,7 +724,7 @@ readfile (
      * always using the GUI.
      */
     if (read_stdin) {
-      mch_msg(_("Vim: Reading from stdin...\n"));
+      mch_msg(_("Nvim: Reading from stdin...\n"));
```
Comment (justinmk) [inline src/nvim/fileio.c]: `.po` files need to be updated

File: src/nvim/normal.c [hunk 2]
```diff
@@ -6934,7 +6934,7 @@ static void nv_esc(cmdarg_T *cap)
         && cmdwin_type == 0
         && !VIsual_active
         && no_reason)
-      MSG(_("Type  :quit<Enter>  to exit Vim"));
+      MSG(_("Type  :quit<Enter>  to exit Nvim"));
```
Comment (justinmk) [inline src/nvim/normal.c]: `.po` files need to be updated

File: src/nvim/fileio.c (hunk 1 above)
Comment (Pyrohh) [inline src/nvim/fileio.c]: done

**Comment by Pyrohh** [comment]
merged

**Pull request closed without merging**